timeboost-types/src/candidate_list.rs

```diff
@@ -18,6 +20,7 @@ struct Inner {
     index: DelayedInboxIndex,
     priority: Vec<SignedPriorityBundle>,
     regular: Vec<Bundle>,
+    dkg: Option<DkgBundle>,
```

Comment:
why a single bundle instead of a vector,
isn't a CandidateList extrapolated from all the DAG vertex from a sailfish round? which may contains multiple dkg dealings?